defaults/main.yml

```diff
@@ -67,3 +67,4 @@ opencraft_all_env_tokens: "{{ OPENCRAFT_GENERATED_ENV_TOKENS | combine( OPENCRAF
 www_data_home_dir: '/var/www'
 opencraft_root_dir: '/var/www/opencraft'
 opencraft_virtualenv_dir: '/var/www/.virtualenvs/opencraft'
+OPENCRAFT_STATIC_FILES_DIR: '{{ opencraft_root_dir }}/build/static'
```

Comment:
@smarnach this is good to know.
I was not sure what was the guideline here, as I had seen both lower and uppercase variables.

I'll change this to lowercase and update the PR.